src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,68 @@
+package lotto.model;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+    private final List<LottoNumber> numbers;
+
+    public Lotto() {
+        this(generateRandomNumbers());
+    }
+
+    public Lotto(int... numbers) {
+        this(Arrays.stream(numbers).boxed().collect(Collectors.toList()));
+    }
+
+    public Lotto(List<Integer> numbers) {
+        checkValidity(numbers);
+        Collections.sort(numbers);
+        this.numbers = numbers.stream().map(LottoNumber::new).collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    public Lotto(List<Integer> numbers) {
        this(numbers.stream().map(LottoNumber::new).collect(Collectors.toList()));
    }
    
    public Lotto(List<LottoNumber> numbers) {
        checkValidity(numbers);
        Collections.sort(numbers);
        this.numbers = numbers;
    }
```
유효성 체크, 필드에 값을 할당등은 주 생성자를 추가한 후 주 생성자가 담당하도록 구현하는 습관을 들여보면 어떨까?